packages/pwa/app/utils/phone-utils.js

```diff
@@ -20,3 +20,14 @@ export const formatPhoneNumber = (value) => {
     }
     return `(${phoneNumber.slice(0, 3)}) ${phoneNumber.slice(3, 6)}-${phoneNumber.slice(6, 10)}`
 }
+/**
+ * Validates given Phone Number
+ * @param {string} - Phone number to be validated
+ * @returns {boolean}  - Result of Validation
+ */
+export const validatePhone = (value) => {
```

Comment:
Hey @agurumurthy96, thanks for opening this PR.

The validation works well for AMER phone numbers but doesn't consider other valid international formats like phone numbers from EMEA.

I think we can use a trusted third party library that takes international format into account, like: https://www.npmjs.com/package/react-phone-number-input#user-content-react-hook-form

What do you think?

Reply:
The reason I didn't want to use any external library was to keep the code light. Maybe I could try finding other regular expressions that includes all use cases. Also, the package that you shared, uses google's libphonenumberjs internally, so if it comes to using a package, maybe we can directly use that instead?

Reply:
i think it’s okay to provide a reasonable default validator, and up to the customer to customize it when needed. Adding a new library makes it harder to customize and adds weight to the javascript bundles. I personally think it might not worth adding a library for this.

Reply:
if a customer needs to validate international formats, I think it's better to let the project implementor to customize it, they could choose to use any library at their freedom.

Reply:
@agurumurthy96 would you mind adding a comment explain which phone number format(s) this regex works? and maybe warn users that this might not work for international formats and they will have to customize this if needed.

Reply:
That's a fair point. 

Adding an example validation with a comment, it's a good option.

Another option, in my opinion, is to not add a validation.
  Ideally, we want to provide the same functionality that the underlying API provides.
  Testing the endpoint [Shoppers Customer](https://developer.commercecloud.com/s/api-details/a003k00000UHvpJAAT/commerce-cloud-developer-centershoppercustomers?tabset-888ee=2) I see that accepts phone numbers of various lengths.
I think that could be something that is provided as a feature of the API, but we'd be dumbing it down for our PWA.

Just another thought, I'm fine with either option.


@agurumurthy96 It seems like your latest changes break the phone validation:
<img width="290" alt="Screen Shot 2021-12-10 at 2 29 15 PM" src="https://user-images.githubusercontent.com/6024102/145581535-c3fd52c5-001c-4047-ad01-6992720f2faf.png">